README.md

```diff
@@ -50,12 +55,23 @@ In your application, declare the angular-google-analytics module dependency.
 <script src="bower_components/angular-google-analytics/dist/angular-google-analytics.js"></script>
 ```
 
-In your application, declare dependency injection like so.
+#### 2- In your application, declare dependency injection:
 
 ```javascript
-angular.module('myModule', ['angular-google-analytics']);
+var myApp = angular.module('myModule', ['angular-google-analytics']);
 ```
 
+#### 3- Set your Google Analytics account and start tracking:
+
+```JavaScript
+myApp.config(function (AnalyticsProvider) {
+   // Add configuration code as desired
+   AnalyticsProvider.setAccount('UU-XXXXXXX-X');  //UU-XXXXXXX-X should be your tracking code
+}).run(['Analytics'], function(Analytics) { });
+```
+Now [angular-google-analytics](https://github.com/revolunet/angular-google-analytics) is ready and is working which means Google Analytics is tracking your page now, but you still need to read rest of documentation to learn more about different configurations and more.
```

Comment:
This can read a bit better by reducing unnecessary words:

"Congratulations! [angular-google-analytics](https://github.com/revolunet/angular-google-analytics) is ready and Google Analytics will track your page views once the application is run."

With my previous comment the purpose of the quick start and the justification for why to read the remainder of the document was placed at the top of this section.